optuna/samplers/_tpe/sampler.py

```diff
@@ -324,14 +324,16 @@ def _split_observation_pairs(
         self, config_vals: List[Optional[float]], loss_vals: List[Tuple[float, float]]
     ) -> Tuple[np.ndarray, np.ndarray]:
 
-        config_vals = np.asarray(config_vals)
-        loss_vals = np.asarray(loss_vals, dtype=[("step", float), ("score", float)])
-
-        n_below = self._gamma(len(config_vals))
-        loss_ascending = np.argsort(loss_vals)
-        below = config_vals[np.sort(loss_ascending[:n_below])]
+        config_values: Union[List[Optional[float]], np.ndarray]
+        config_values = np.asarray(config_vals)
+        loss_values: Union[List[Optional[float]], np.ndarray]
+        loss_values = np.asarray(loss_vals, dtype=[("step", float), ("score", float)])
```

Comment:
Sorry for my previous inappropriate comment. Can be removed the type annotations. 
```suggestion
        config_values = np.asarray(config_vals)
        loss_values = np.asarray(loss_vals, dtype=[("step", float), ("score", float)])
```